source/integrations-guide/integrations-guide-index.rst

```diff
@@ -6,370 +6,127 @@ Integrations Guide
+Learn more about :doc:`Mattermost slash commands </integrations-guide/slash-commands>`.
 
-Pre-built slash commands
+Built-In Slash Commands
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-**Technical complexity: No-code**
-
-These out-of-the-box commands enable command line interaction with users, channels, and conversations. Examples of pre-built slash commands include ``/invite`` to add teammates to a channel, ``/header`` to set a channel description, or ``/call`` to start or join a call in a channel.
+**Technical complexity:** :ref:`No-code <no-code>`
 
-Learn more about :doc:`pre-built slash commands </integrations-guide/built-in-slash-commands>`.
+Out-of-the-box commands enable command line interaction with users, channels, and conversations.
 
-Custom slash commands
-^^^^^^^^^^^^^^^^^^^^^
+Custom Slash Commands
+^^^^^^^^^^^^^^^^^^^^^^
 
-**Technical complexity: Low-code**
+**Technical complexity:** :ref:`Low-code <low-code>`
 
-Slash commands are similar to an outgoing webhooks, but instead of listening for keywords in a channel they are triggered by users explicitly typing a preconfigured command. Using slash commands is similar to using a command line within a channel. This means if you type in a slash command it will execute without posting a message, whereas an outgoing webhook is only triggered by posted messages. The external service then processes the request and can return a response, such as plain text, rich message content, interactive buttons or forms, directly into the channel.
-
-Mattermost's slash command format is Slack compatible, so you can easily migrate your commands from Slack.
-
-Learn more about `building custom slash commands <https://developers.mattermost.com/integrate/slash-commands/custom/>`_.
-
-**Examples**
-
-Here are some example use cases for slash commands in Mattermost:
-
-- **Project management**  
-   Typing ``/jira create`` opens an interactive dialog where you can file a new Jira issue directly from Mattermost, including fields like summary, description, and priority.
-
-- **Incident response**  
-   Using ``/pagerduty trigger`` can open a form to start a new incident, notify on-call responders, and post details back into the channel so the team can coordinate.
-
-- **Knowledge retrieval**  
-   A command like ``/docs search authentication`` queries your documentation system and returns a list of relevant articles as interactive message attachments with links.
+You can create custom slash commands that run preconfigured commands that can return a response, such as plain text, rich message content, interactive buttons or forms, directly into a channel.
```

Comment:
Add as new paragraph to this sub section: 

Mattermost's slash command format is Slack compatible, so you can easily migrate your commands from Slack.